src/commands/DiffCommand.js

```diff
@@ -38,4 +38,15 @@ export default class DiffCommand extends Command {
       }
     });
   }
+
+  runValidations() {
+    super.runValidations();
+    try {
+      GitUtilities.hasCommit();
```

Comment:
Oh never mind the previous comment, I see what you are doing.

Could you actually move this to `initialize()` and create a version of `hasCommit` that doesn't throw and just returns a boolean?